DEVELOPING.md

```diff
@@ -186,7 +186,7 @@ You'd need alternative DSL import since you can't rely on sbt package object.
 
 ```scala
 // for slash syntax
-import sbt.SlashSyntax0.given
+import sbt.SlashSyntax0.*
```

Comment:
See https://www.scala-sbt.org/2.x/docs/en/changes/migrating-from-sbt-1.x.html#changing-buildsbt-dsl-to-scala-3x

```scala
import sbt.SlashSyntax0.{ given, * }
```

should work?

Reply:
It works, but it makes migration of plugins more complicated...

The downside is that "old style implicits" are not idiomatic scala3. Also my idea was to move to givens in sbt 2.1, but thinking about it, that would be a big breaking change that would require all plugins to be republished...

I will close this